src/components/FilterPanel/index.tsx

```diff
@@ -203,16 +239,25 @@ class FilterPanel extends React.Component<Props, States> {
       selectedValues,
       availableValues,
       filterInputState,
-      setFilterInputState,
     } = this.props;
+    const { value } = this.state;
     const entityName = attrTabValue[selectedEntity];
     const cards = attributes.reduce((acc, current) => {
       if (current.cardinality < 15 && current.cardinality !== null) {
         acc.push(current.name);
       }
       return acc;
     }, []);
-
+    const currentValue = [];
```

Comment:
Where did you use `currentValue`?

Reply:
That was leftover code I forgot to remove